.travis.yml

```diff
@@ -0,0 +1,46 @@
+---
+sudo: false
+dist: trusty
+language: ruby
+cache: bundler
+before_install:
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
+script:
+  - 'bundle exec rake $CHECK'
+  - 'bundle exec rspec spec/'
+bundler_args: --without system_tests
+rvm:
+  - 2.5.0
+env:
+  global:
+    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
+matrix:
+  fast_finish: true
+  include:
+    -
+      env: CHECK="syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop"
```

Comment:
I don't think the Puppet version matters for this. The important things are probably testing Ruby 2.3 or 2.4, and 2.5. Doesn't look like we test with Ruby 2.5 here.

Reply:
Not from this line? https://github.com/puppetlabs/puppetlabs-ruby_task_helper/pull/1/files/d1e4c4c8183924ee096971896e8d8e41e63bf438#diff-354f30a63fb0907d4ad57269548329e3R17

Reply:
It'll use Ruby 2.5, but it's not running specs.